app/sessionStore.js

```diff
@@ -59,7 +59,14 @@ module.exports.saveAppState = (payload, isShutdown) => {
         wndPayload.tabs = wndPayload.tabs.filter((tab) => !tab.isPrivate)
       })
     } else {
-      delete payload.perWindowState
+      // Reset the window state except for the ui data
+      payload.perWindowState.forEach((wndPayload) => {
```

Comment:
Since it will be an empty array I don't see any problem if it is not unset, because on the next initialization it will be set to empty array again. Let me know if you want me to add check tho